tests/signals/tests.py

```diff
@@ -317,3 +318,16 @@ class Created(models.Model):
 
         Created()
         self.assertEqual(received, [])
+
+    def test_register_model_class_senders_immediately(self):
+        """
+        Test that model signals registered with model classes as senders
```

Comment:
Just state the expected behavior instead of adding a "Test that..." preamble to all messages (the test suite isn't perfect about this yet).